### ISSUE TITLE
Add 60fps support

### ISSUE THREAD

**Issue by githubcatw** [google-ar/arcore-unity-sdk#554]
Add 60fps support, like ARKit, even if it will be on flagships only. It might also increase tracking accuracy.

**Comment by andrelevi** [comment]
Agreed. Yes, we can make the app render at 60 fps even though ARCore is locked at 30hz. But we want 60hz pose data and camera image updates to match ARKit.
